.editorconfig

```diff
@@ -0,0 +1,11 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Is there any need for final newline insertion? I think that they should be removed and this setting should be set to `false`. If anyone agrees, I can create a PR and then we could merge it.

Reply:
I think that one is very subjective.
The example on editorconfig.org has this option set to `true`, and I have been working on a few project with it, without any inconvenience.

So I don't see the advantage of changing this option to `false`.
Furthermore changing that will bring an update of almost all code files, which is IMHO not necessary when things are already consistent.

Reply:
Okay, I don’t argue, though I personally prefer keep it `false`. And I know that [editorcong.org](editorconfig.org] has it that way and IMO it is the main reason everyone default to that. And I don’t any advantage of keeping a final newline either.

Anyway, keep it the way it is. :)